src/main.cpp

```diff
@@ -142,6 +148,7 @@ void initialize() {
             pros::delay(10);
         }
     });
+	// the color sort task can't be put here due to some issue with defining the color signatures?
```

Comment:
define the sigs (237 ish) at the top after the constructors so you can put the color sort task up here

Reply:
Yeah I tried that and I got an error saying that it can't hold the value or something?

Reply:
hm, I'll see what I can figure out